### PR TITLE
add changelog and bump version to 0.8.2

### PR DESCRIPTION
**Pull request by davitol**
Add relevant Changelog items and bump version to 0.8.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+
+## [0.8.2] - (final release date unknown)
+
+### Fixed
+
+- Adjust guest sharing dropdown text deleting the extra (guest) in brackets  - [#337](https://github.com/owncloud/guests/pull/337)
```
Comment (patrickjahns) [inline CHANGELOG.md]: that's a changed part - I would rephrase it as "Removed guest label in drop down to adjust with core display changes"

**Comment by phil-davis** [comment]
drone fail is some "random" agent communication error: https://drone.owncloud.com/owncloud/guests/921/154
```
Error response from daemon: unexpected EOF
```
